opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -12,26 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import opentelemetry.resources as resources
+import typing
 
+LabelValue = typing.Union[str, bool, int, float]
+Labels = typing.Optional[typing.Dict[str, LabelValue]]
 
-class Resource(resources.Resource):
-    def __init__(self, labels):
+
+class Resource:
+    def __init__(self, labels: Labels):
         self._labels = labels
 
     @staticmethod
-    def create(labels):
+    def create(labels: Labels) -> "Resource":
+        if not labels:
+            return _EMPTY_RESOURCE
         return Resource(labels)
 
+    @staticmethod
+    def create_empty() -> "Resource":
+        return _EMPTY_RESOURCE
```

Comment:
one thing about Resource is it's not exactly immutable right now, since the labels property returns the underlying container as an optimization. 

Since we don't know how labels will be used anyway, should we just create a copy of the labels for now? That'll make sure the code above works.

Otherwise, all the default references to the empty Resource should be replaced with a constructor for a blank resource.